destroy_cluster.sh

```diff
@@ -106,7 +106,10 @@ if [[ $FORCE == true ]]; then
     fi
 else
     echo Destroying cluster using openshift-install
-    "$installer" --log-level=debug destroy cluster --dir "${TMP_DIR:-$ARTIFACT_DIR}"
+    timeout 900 "$installer" --log-level=debug destroy cluster --dir "${TMP_DIR:-$ARTIFACT_DIR}"
```

Comment:
I could probably add a var instead of hardcoding 900, but I thought 15 min max was a sane assumption.